cli/providers/aws/logs/logs.go

```diff
@@ -0,0 +1,31 @@
+package logs
+
+import (
+	"fmt"
+	logger "log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/cloudwatchlogs"
+	"github.com/pchico83/i2kit/cli/schemas/service"
+)
+
+//Logs prints the logs asociatted to a given instance
+func Logs(s *service.Service, startTime *int64, config *aws.Config, log *logger.Logger) error {
+	svc := cloudwatchlogs.New(session.New(), config)
+	input := &cloudwatchlogs.FilterLogEventsInput{
+		LogGroupName: aws.String(fmt.Sprintf("i2kit-%s", s.Name)),
+		StartTime:    startTime,
+	}
+	logEvents, err := svc.FilterLogEvents(input)
+	if err != nil {
+		return nil
```

Comment:
I think we should return the error  so we can tell what went wrong

Reply:
Same as before, the error is not helpful to the caller in this case.
I prefer not to return something which is gonna be always ignored.